lib/commands/actions.js

```diff
@@ -119,7 +119,7 @@ commands.isLocked = async function () {
 };
 
 commands.unlock = async function () {
-  return await androidHelpers.unlock(this.adb);
+  return await androidHelpers.unlockWithHelperApp(this.adb);
```

Comment:
Does this mean running the `unlock` command mid-session cannot use the other methods? Might it not be possible to allow it, if they are passed in through the caps at the beginning of the session?